app/org/apache/spark/deploy/history/SparkFSFetcher.scala

```diff
@@ -237,7 +237,11 @@ class SparkFSFetcher(fetcherConfData: FetcherConfigurationData) extends Elephant
               null
             }
           } else {
-            val logFilePath = new Path(logPath + "_1.snappy")
+            var sparkLogExt = fetcherConfData.getParamMap.get(SPARK_LOG_EXT)
```

Comment:
Not too familiar with the classes here, but `get()` doesn't support a 2nd argument that is the default value?